src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.css

```diff
@@ -1,6 +1,11 @@
 @import '../../styles/colors-v2.css';
 @import '../../styles/spacing.css';
 
+.autocomplete-input {
```

Comment:
I'm confused about the changes to this file. Why are there so many CSS changes for a new sub-component?

Reply:
The selectors weren't alphabetized. I was under the impression that that's the pattern we typically follow, and so I alphabetized all of the CSS rules for us as part of my work